packages/react-composites/src/composites/CallComposite/CallControls.tsx

```diff
@@ -36,7 +48,7 @@ export const CallControls = (props: GroupCallControlsProps): JSX.Element => {
       <CameraButton
         {...cameraButtonProps}
         onToggleCamera={() => {
-          return cameraButtonProps.onToggleCamera().catch((e) => console.log(e));
+          return cameraButtonProps.onToggleCamera(localVideoViewOption).catch((e) => console.log(e));
```

Comment:
If we always need a localVideoViewOption, then it is better to pass localVideoViewOption into Component then call it from inside

onToggleCamera() of function should always stay the same as what we provide from handler, including parameters

Reply:
`
<CameraButton 
 {...cameraButtonProps}
 localVideoViewOption={localVideoViewOption}
/>
`

Then onToggleCamera in CameraButton should be able to receive it (you'll need to change internal implementation of CameraButton a little bit)